docker-compose.yml

```diff
@@ -76,14 +76,13 @@ services:
       - INGRESS_VALIDTOPICS=testareno,advisor,buckit,compliance #if you test a different topic, add it here
       - INGRESS_INVENTORYURL=inventory-web:8081/api/inventory/v1/hosts
       - OPENSHIFT_BUILD_COMMIT=woopwoop
-      - INGRESS_PORT=8080
       - INGRESS_MINIODEV=true
       - INGRESS_MINIOACCESSKEY=$MINIO_ACCESS_KEY
       - INGRESS_MINIOSECRETKEY=$MINIO_SECRET_KEY
       - INGRESS_MINIOENDPOINT=minio:9000
       - INGRESS_MAXSIZE=104857600 # 100 MB
     ports:
-      - 8080:8080
+      - 8080:3000
```

Comment:
But this conflicts with our backend port. We don't care about metrics from ingress. Did this start failing for you with your recent image pull we were discussing yesterday?

Reply:
Maybe you meant `3000:8080` ?

Reply:
This should be correct. The order is `hostPort:containerPort`.
```
hostPort = 8080
containerPort = 3000
```

Or, are you saying that this conflicts within a pod?

Reply:
No, sorry, you're right. This is silly that's it's opposite as the related `-p` flag when running a one-off container. https://docs.docker.com/compose/compose-file/#ports